core/modules/dashboard/includes/block.news.inc

```diff
@@ -40,13 +40,30 @@ class DashboardNewsBlock extends Block {
    */
   public function getContent() {
     $news = $this->getNews();
+    $days_before_news_is_old = 14;
+
     $links = array();
     if ($news) {
       foreach ($news as $key => $notifications) {
-        $links[] = '<p>' . l(
-            $notifications['title'],
-            $notifications['url']
-          ) . ' - ' . filter_xss($notifications['summary']) . '</p>';
+        $notification_date = format_date($notifications['timestamp'], 'custom', 'M j, Y');
+        $date_classes = array('class' => array('notification-date'));
+        $is_new = (time() - $notifications['timestamp']) < $days_before_news_is_old * 24 * 60 * 60;
```

Comment:
```suggestion
        // A notification is considered old, if 14 days (1209600 seconds) have
        // lapsed.
        $is_new = (time() - $notifications['timestamp']) < 1209600;
```

Reply:
I love it!

Reply:
Looks good

Reply:
Suggestion for better readability:

```
        // A notification is considered new, if no more that 14 days
        // (1209600 seconds) have lapsed.
```

Reply:
Done: [`f57e8a0` (#3633)](https://github.com/backdrop/backdrop/pull/3633/commits/f57e8a0be11a952070cb749eac46a8d30e621d6f)

@olafgrabienski please see our [coding standards re inline comments](https://docs.backdropcms.org/doc-standards#general):
> - Lines containing comments (including docblocks) must wrap **as close to 80 characters as possible** without going over...

Reply:
Fair enough! In my opinion it's good to interpret the standard formally less strict in such a case (readability). But that discussion should probably happen in another place.